openedx/core/djangoapps/programs/models.py

```diff
@@ -33,6 +30,8 @@ class ProgramsApiConfig(ConfigurationModel):
             "This value is required in order to enable the Studio authoring interface."
         )
     )
+
+    # TODO: The property below is obsolete. Delete at the earliest safe moment
```

Comment:
Is there a ticket (e.g., dev story) for this? Maybe you could include the ticket number in the comment.